### PR TITLE
Bump `aboutlibraries` to `11.2.1`

### PR DESCRIPTION
**Pull request by wzieba**
## Description

This PR updates `aboutlibraries` to address https://github.com/Automattic/pocket-casts-android/security/dependabot/19

## Testing Instructions
Install automotive app, open Settings and then Acknowledgements. Verify that a list of libraries is present.

## Screenshots or Screencast 

image

## Checklist
- [ ] If this is a user-facing change, I have added an entry in CHANGELOG.md
- [ ] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [ ] I have considered whether it makes sense to add tests for my changes
- [ ] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [ ] Any jetpack compose components I added or changed are covered by compose previews
- [ ] I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.
 
#### I have tested any UI changes...

- [ ] with different themes
- [ ] with a landscape orientation
- [ ] with the device set to have a large display and font size
- [ ] for accessibility with TalkBack

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: app/lint-baseline.xml [hunk 1]
```diff
@@ -56,7 +56,7 @@
         id="InvalidPackage"
         message="Invalid package reference in library; not included in Android: `java.awt`. Referenced from `com.sun.jna.Native.AWT`.">
         <location
-            file="$GRADLE_USER_HOME/caches/transforms-3/61f2ace0377fec944d0761aa7634924b/transformed/jna-5.13.0/jars/classes.jar"/>
+            file="$GRADLE_USER_HOME/caches/transforms-3/a5c0a6fe0233ae59f2dafd8082c122a9/transformed/jna-5.13.0/jars/classes.jar"/>
```
Comment (wzieba) [inline app/lint-baseline.xml:59]: Here's what I meant @MiSikora in our [discussion](https://github.com/Automattic/pocket-casts-android/pull/2283/files#r1624254613).

Comment (MiSikora) [reply app/lint-baseline.xml:59]: I see. It might be worth in that case to add `InvalidPackage` as an ignored issue to `lint.xml` file. What do you think?

Comment (wzieba) [reply app/lint-baseline.xml:59]: I think it's a good idea! I'll do this in scope of #2301 . The other baseline record that uses `$GRADLE_USER_HOME`, `ObsoleteLintCustomCheck` is resolved there (fixed).

**Pull request merged**